examples/tutorials/code_2_hyperband_checkpoint.py

```diff
@@ -0,0 +1,392 @@
+"""
+====================
+Checkpointing trials
+====================
+
+.. hint::
+
+    In short, you should use "{experiment.working_dir}/{trial.hash_params}" to set the path of
+    the checkpointing file.
+
+When using multi-fidelity algorithms such as Hyperband it is preferable to checkpoint the trials
+to avoid starting training from scratch when resuming a trial. In this tutorial for instance,
+hyperband will train VGG11 for 1 epoch, pick the best candidates and train them for 7 more epochs,
+doing the same again for 30 epoch, and then 120 epochs. We want to resume training at last epoch
+instead of starting from scratch.
+
+Oríon provides a unique hash for trials that can be used to define the unique checkpoint file
+path: ``trial.hash_params``. This can be used with the Python API as demonstrated in this example
+or with :ref:`commandline_templates`.
+
+With command line
+-----------------
+
+The example below is based on the Python API solely. It is also possible to do checkpointing
+using the command line API. To this end, your script should accept an argument for the checkpoint
+file path. Suppose this argument is ``--checkpoint``, you should call your script with the
+following template.
+
+::
+
+    orion hunt -n <exp name>
+        ./your_script.sh --checkpoint '{experiment.working_dir}/{trial.hash_params}'
+
+Your script it reponsible to take this checkpoint path, resume from checkpoints or same
```

Comment:
Grammar/Spelling: "Your script it responsible..." -> "Your script is responsible..."